apps/react-workshop/src/Footer.stories.tsx

```diff
@@ -107,7 +107,12 @@ export const CustomContent = () => {
                   <Footer.Separator />
                   <Footer.Item>
                     {element.url ? (
-                      <a href={element.url} target='_blank' rel='noreferrer'>
+                      <a
+                        href={element.url}
+                        target='_blank'
+                        rel='noreferrer'
+                        className='_iui3-anchor'
```

Comment:
This is not acceptable. The class names are an implementation detail that should not be exposed to the user. (The "_" prefix indicates that it's considered private)

Reply:
Okay, then I'll replace the `<a>` here with `Anchor`.

Reply:
Removed that `className` in the latest [commit.](https://github.com/iTwin/iTwinUI/pull/1915/commits/45c6861c0477f194b1f2e3955493ea8bdf78321b)